percy/render/recipe_parser.py

```diff
@@ -224,15 +235,58 @@ def __str__(self) -> str:
         Renders the Node as a string. Useful for debugging purposes.
         :return: The node's value, as a string
         """
-        return str(self.value)
+        value = self.value
+        if self.is_comment():
+            value = "Comment node"
+        return (
+            f"Node: {value}\n"
+            f"  - Comment:      {self.comment}\n"
```

Comment:
```suggestion
            f"  - Comment:      {self.comment!r}\n"
```
using `!r` could potentially help if dealing with leading/trailing whitespaces.